src/main/java/frc/robot/commands/visionCommands/alignScore.java

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.visionCommands;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.RunCommand;
+import frc.robot.Constants.OIConstants;
+import frc.robot.Constants.VisionConstants;
+import frc.robot.Robot;
+import frc.robot.RobotContainer;
+
+public class alignScore extends Command {
+  /** Creates a new alignScore. */
+  private double angle;
+
+  public alignScore() {
+    addRequirements(RobotContainer.swerveDrive);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    angle = RobotContainer.limeSubsystem.getHorizontalAngle();
+    SmartDashboard.putNumber("ANGLE VALUE IN ALIGN SCORE", angle);
+    SmartDashboard.putNumber("controller x, up down", RobotContainer.getDriverController().getLeftY());
+    SmartDashboard.putNumber("controller x left right", RobotContainer.getDriverController().getLeftX());
+    SmartDashboard.putNumber("DEADBAND", -MathUtil.applyDeadband(RobotContainer.getDriverController().getLeftY(), OIConstants.kDriveDeadband));
+    if (angle < -VisionConstants.threshold || angle > VisionConstants.threshold){
+      RobotContainer.swerveDrive.drive(
+                -MathUtil.applyDeadband(RobotContainer.getDriverController().getLeftY(), OIConstants.kDriveDeadband),
```

Comment:
to reduce redundancy you can make something like
```java
public double getXSpeed() {
  return -MathUtil.applyDeadband(m_driverController.getLeftY(), OIConstants.kDriveDeadband)
}
```

in robot container because we use this multiple palces

Reply:
rather than using getDriverController